ICAFIX/scripts/functionhighpassandvariancenormalize.m

```diff
@@ -32,94 +34,125 @@ function functionhighpassandvariancenormalize(TR,hp,fmri,WBC,varargin)
 
 % Authors: M. Glasser and M. Harms
 
-CIFTIMatlabReaderWriter=getenv('FSL_FIX_CIFTIRW');
-if (~isdeployed)
-  % addpath does not work and should not be used in compiled MATLAB
-  fprintf('Adding %s to MATLAB path\n', CIFTIMatlabReaderWriter);
-  addpath(CIFTIMatlabReaderWriter);
-end
-
 %% Defaults
-dovol = 1;
+dovol = true;
 regstring = '';
 pdflag = false;  % Polynomial detrend
 pdstring = 'pd';  % Expected string at start of HP variable to request a "polynomial detrend"
-hpstring = '';
-volhpwishart = 2;
+volwishart = 2;
+ciftiwishart = 3;
+VN = 1; %first iteration uses 1 for dimensionality
+iters = -1; %iterate until new convergence criteria
```

Comment:
Where do these "new convergence criteria" come from?  Bit more in the usage/help for `icaDim.m` on this parameter would help.

Reply:
Ad-hoc, as far as I know.  There may not be an existing description outside of the icaDim code that implements it.

Reply:
Where do the "new convergence criteria" come from?  
"Iterate until convergence", while vague (doesn't explain how convergence is defined, which isn't appropriate for here anyway) would be a simpler and clearer comment, IMO.

Reply:
icaDim currently implements two modes of convergence testing.

Reply:
So, I think "%iterate until convergence using new algorithm" better captures what is being implied here.

Reply:
"new algorithm" implies the computation is different, "criteria" is a correct term for simply testing whether to stop, without changing the computation.

Reply:
Why don't we just duplicate the nice language you added to `icaDim.m`:
"iterate until the average of the dimensionality history doesn't change much"